.github/workflows/zizmor.yml

```diff
@@ -14,13 +15,13 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: Download zizmor
-        run: cargo install zizmor
+      - name: Install the latest version of uv
```

Comment:
why pin
and what was wrong with cargo install ?

Reply:
I don't have to pin if you don't want to. It just makes it so it stays on the same version no matter what (I think). It is suggested by zizmor `--pedantic`. 

`cargo install` takes longer than uv does because it complies the binary. It is using exactly the same binary. Again I could `cargo install`, but it seems pointless. This is the suggested way to do this on the zizmor site https://woodruffw.github.io/zizmor/usage/#use-in-github-actions. It actually suggests using the SARIF format and uploading the results of a run to GitHub, but I don't see what the point of that is.

Reply:
> It actually suggests using the SARIF format and uploading the results of a run to GitHub, but I don't see what the point of that is.

This is for integration with GitHub Advanced Security. Uploading the SARIF file will make the results available in the "Security" tab of this repository, under code scanning alerts. It should also post comments on PRs that introduce regressions. See, for example: https://github.com/Homebrew/brew/pull/18986